pubsub/google/cloud/pubsub_v1/subscriber/_protocol/streaming_pull_manager.py

```diff
@@ -213,7 +213,7 @@ def maybe_pause_consumer(self):
             if self._consumer is not None and not self._consumer.is_paused:
                 _LOGGER.debug("Message backlog over load at %.2f, pausing.", self.load)
                 self._consumer.pause()
-                print('paused')
+                
```

Comment:
Double line spacing between class methods, I think this is what caused the lint check to fail - just run `black` on it and should be fixed.

Reply:
I tried `nox -s lint` with the proposed change. This file still failed, without any useful description.